packages/plugin-core/src/test/suite-integ/DoctorCommand.test.ts

```diff
@@ -39,11 +39,14 @@ suite("DoctorCommandTest", function () {
     onWSInit(async () => {
       const testFile = path.join(root.name, "vault", "bond2.md");
       fs.writeFileSync(testFile, "bond", { encoding: "utf8" });
+      const testFile2 = path.join(root.name, "vault", "bond3.md");
+      fs.writeFileSync(testFile2, "bond3", { encoding: "utf8" });
```

Comment:
nitpick: when testing code, we have a convention of adding "bond" to output values we want to debug - its easy to find in logs but please don't commit as it makes it harder to grep

Reply:
ah. didn't know that. thanks for letting me know.
do you want me to change the one that was already there?